packages/vue-use-tippy/tsconfig.json

```diff
@@ -6,7 +6,7 @@
 		"declarationDir": "dist",
 		"esModuleInterop": true,
 		"experimentalDecorators": true,
-		"lib": ["ESNext", "ES2019"],
+		"lib": ["ESNext", "ES2019", "DOM"],
```

Comment:
when removing storybook, the DOM lib became EXPLICITLY needed .. because vue-use-tippy relies on the `Element` type which is a DOM lib type